core/commands.py

```diff
@@ -151,3 +154,32 @@ def fill_choices(self, Model):
             self.log(" - done in {:.3f}s.".format(end_field - start_field))
         end = time.time()
         self.log("  done in {:.3f}s.".format(end - start))
+
+
+class UpdateTableFileCommand:
+
+    def __init__(self, table, file_url):
+        self.table = table
+        self.file_url = file_url
+        self.hasher = hashlib.sha512()
+        self.file_size = 0
+
+    def process_file_chunks(self):
+        # TODO get chunks from settings
+        for chunk in stream_file(self.file_url, chunk_size=256):
```

Comment:
256 bytes é muito pouco, o que ocasiona um número muito grande e iterações (e deixa o código mais lento). Nesse caso vale a pena ocupar um pouco mais de memória e ter bem menos iterações:

```suggestion
        for chunk in stream_file(self.file_url, chunk_size=64 * 1024):
```

Reply:
Já foi resolvido no commit que joga esse valor para o settings.